Makefile

```diff
@@ -43,16 +46,25 @@ VERSION ?= main
 
 TAG_LATEST ?= false
 
+ifeq ($(TAG_LATEST), true)
+	IMAGE_TAGS ?= $(IMAGE):$(VERSION) $(IMAGE):latest
+else
+	IMAGE_TAGS ?= $(IMAGE):$(VERSION)
+endif
+
 # The version of restic binary to be downloaded for power architecture
 RESTIC_VERSION ?= 0.9.6
 
 CLI_PLATFORMS ?= linux-amd64 linux-arm linux-arm64 darwin-amd64 windows-amd64 linux-ppc64le
-CONTAINER_PLATFORMS ?= linux-amd64 linux-ppc64le linux-arm linux-arm64
-MANIFEST_PLATFORMS ?= amd64 ppc64le arm arm64
+BUILDX_PLATFORMS ?= linux/amd64,linux/arm64,linux/arm/v7,linux/ppc64le
```

Comment:
does `buildx` also support building windows containers should we need to build one in the future?

Reply:
I don't have any experience working with Windows container images, but is that really needed? Windows can run `linux/amd64` (in the event the Windows arch is x86_64) without any issues (as far as I know), so there is no need for a Windows based image.